contrib/cirrus/runner.sh

```diff
@@ -65,6 +65,10 @@ _run_validate() {
     podmanmake validate-local BUILDTAGS="$BUILDTAGS"
 }
 
+_run_doccheck() {
```

Comment:
Will this do? New runner.sh target, invoked in CI, with a name unambiguously different from the Makefile target.

Reply:
Works for me.

RFC: @cevich  I don’t think c/image tests _need_ to run this, but there might well be something I’m missing about keeping the two test systems consistent.

Reply:
Yes, keeping the `skopeo` testing task YAML 100% the same between the two repos is important since it simplifies the burden of maintaining the necessary duplication.  But I'm fine with dropping the extra validate call in that task here (and in c/image).  I was slightly afraid the extra validation might be necessary to confirm golang is still happy (in c/image), after grafting c/skopeo code in.  But it sounds like you feel this is unnecessary, and I'm fine with removing it.  That way the docs testing only happens in skopeo CI, keeping things simple.